Tools/autotest/quadplane.py

```diff
@@ -103,6 +103,40 @@ def get_disarm_delay(self):
     def set_autodisarm_delay(self, delay):
         self.set_parameter("LAND_DISARMDELAY", delay)
 
+    def test_motor_mask(self):
+        """Check operation of output_motor_mask"""
+        if not(int(self.get_parameter('Q_TILT_MASK')) & 1):
+#        or (int(self.get_parameter('Q_TAILSIT_MOTMX')) & 1)
```

Comment:
Dead code

Reply:
https://github.com/ArduPilot/ardupilot/pull/10401 would add a new parameter: Q_TAILSIT_MOTMX
that would also result in invoking output_motor_mask

Reply:
Good-o.  We ordinarily don't like dead code, but *shrug*.